src/components/FeatureViews/Search/SearchResults.js

```diff
@@ -223,40 +179,40 @@ const SearchResults = (props) => {
                                 }
                             </Grid>
                             <Grid container spacing={16} direction={"row"}>
-                                {searchState.primaryFilter === SEARCH_ACCOUNTS ?
-                                    accounts.map((account) => (
-                                        <Grid item
-                                              key={account.user_id}
-                                            sm={3}>
-                                            <AccountsCards user={account} key={account.user_id} />
-                                        </Grid>
-                                    ))
-                                    :
-                                    account_num_results > 0 && searchState.primaryFilter !== SEARCH_COURSES?
-                                        accounts.slice(start["account"], end["account"]).map((account) => (
-                                            <Grid item
-                                                  key={account.user_id}
-                                                sm={3}>
-                                                <AccountsCards user={account} key={account.user_id} />
-                                            </Grid>
-                                        ))
-                                        :
-                                        ""}
+                                {
+                                    accountResultsNum > 0 && searchState.primaryFilter !== SEARCH_COURSES ?
```

Comment:
Can replace ternary with an alternate value of `""` with an `&&`, in JSX (if the condition is false, nothing will be rendered)